llpc/test/shaderdb/relocatable_shaders/TriangleVs_CheckNoteSectionForCacheHash.spvasm

```diff
@@ -0,0 +1,80 @@
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -gfxip=9 -unlinked -enable-relocatable-shader-elf -v -add-hash-to-elf %s | FileCheck --check-prefixes=CHECK %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: AMDLLPC SUCCESS
+; END_SHADERTEST
+
+; CHECK: .note (size = 80 bytes)
```

Comment:
It is convention to place the CHECK between the `BEGIN_SHADERTEST` and `END_SHADERTEST`.  It makes it clearer which checks belong with which tests when you have multiple tests using the same source file.

Reply:
done